iaso/api/mobile/org_units.py

```diff
@@ -30,7 +32,34 @@ def get_page_number(self, request):
         return int(request.query_params.get(self.page_query_param, 1))
 
 
-class MobileOrgUnitSerializer(ModelSerializer):
+class ReferenceInstancesFilter(django_filters.rest_framework.FilterSet):
+    last_sync = django_filters.IsoDateTimeFilter(field_name="updated_at", lookup_expr="gte")
```

Comment:
This works with `django.utils.dateparse`:

- [`class IsoDateTimeFilter`](https://github.com/carltongibson/django-filter/blob/4a618e080ac904cb9bbcd0066878d3e06014437e/django_filters/filters.py#L289-L300)
- [`class IsoDateTimeField`](https://github.com/carltongibson/django-filter/blob/4a618e080ac904cb9bbcd0066878d3e06014437e/django_filters/fields.py#L125C7-L146)

So you can use various formats:

```python
In [1]: from django.utils.dateparse import parse_datetime

In [2]: parse_datetime("2023-09-26T17:21:22.921692Z")
Out[2]: datetime.datetime(2023, 9, 26, 17, 21, 22, 921692, tzinfo=<UTC>)

In [3]: parse_datetime("2021-09-26T17:21:22Z")
Out[3]: datetime.datetime(2021, 9, 26, 17, 21, 22, tzinfo=<UTC>)

In [4]: parse_datetime("2021-09-26T17:21:22")
Out[4]: datetime.datetime(2021, 9, 26, 17, 21, 22)

In [5]: parse_datetime("2021-09-26T17:21")
Out[5]: datetime.datetime(2021, 9, 26, 17, 21)
```